api/models/model.py

```diff
@@ -596,12 +596,14 @@ def model_config(self):
                     model_config = app_model_config.to_dict()
                 else:
                     model_config["configs"] = override_model_configs
+
             else:
                 app_model_config = (
                     db.session.query(AppModelConfig).filter(AppModelConfig.id == self.app_model_config_id).first()
                 )
                 if app_model_config:
                     model_config = app_model_config.to_dict()
+                    model_config["model_dict"] = app_model_config.model_dict
```

Comment:
The property is already present in the 'model' object when we call the 'to_dict()' method. It appears that this issue cannot be resolved.

Reply:
Got it, may be give me some tip?

Reply:
I understand the issue now, as `model_config` corresponds to `model_config_fields`, not `simple_model_config_fields`. Perhaps this approach would work: redefine a return dictionary.
```
 result = {}
 result["model_dict"] = model_config["model"]
 result["pre_prompt"] = ""
 return result
```
What do you think？